packages/contracts/deploy-config/sepolia.json

```diff
@@ -34,5 +34,7 @@
   "colosseumDummyHash": "0xe3c0fb45c84ee6608b3ee3a7016c505f46ff23736038a4344abf62156e2b21be",
   "colosseumMaxTxs": 25,
   "colosseumSegmentsLengths": "6,5",
-  "numDeployConfirmations": 1
+  "numDeployConfirmations": 1,
+  "securityCouncilNumConfirmationRequired": 1,
```

Comment:
In production, it's safer to set it to 2/3 or higher.

Reply:
Yes, this should be a policy decision.